packages/block-library/src/missing/edit.native.js

```diff
@@ -1,43 +1,106 @@
 /**
  * External dependencies
  */
-import { View, Text } from 'react-native';
+import { Platform, View, Text, TouchableWithoutFeedback } from 'react-native';
 
 /**
  * WordPress dependencies
  */
-import { Icon } from '@wordpress/components';
+import { BottomSheet, Icon } from '@wordpress/components';
 import { withPreferredColorScheme } from '@wordpress/compose';
 import { coreBlocks } from '@wordpress/block-library';
 import { normalizeIconObject } from '@wordpress/blocks';
 import { Component } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 
 export class UnsupportedBlockEdit extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { showHelp: false };
+	}
+
+	toggleSheet() {
+		this.setState( {
+			showHelp: ! this.state.showHelp,
+		} );
+	}
+
+	renderHelpIcon() {
+		return <TouchableWithoutFeedback
+			accessibilityLabel={ __( 'Help icon' ) }
+			accessibilityRole={ 'button' }
+			accessibilityHint={ __( 'Tap here to show help' ) }
+			onPress={ this.toggleSheet.bind( this ) }
+		>
+			<View style={ styles.helpIconContainer } >
+				<Icon
+					className="unsupported-icon-help"
+					label={ __( 'Help icon' ) }
+					icon="editor-help"
+				/>
+			</View>
+		</TouchableWithoutFeedback>;
+	}
+
+	renderSheet( title ) {
+		const { getStylesFromColorScheme } = this.props;
+		const infoTextStyle = getStylesFromColorScheme( styles.infoText, styles.infoTextDark );
+		const infoTitleStyle = getStylesFromColorScheme( styles.infoTitle, styles.infoTitleDark );
+		const infoDescriptionStyle = getStylesFromColorScheme( styles.infoDescription, styles.infoDescriptionDark );
+
+		// translators: %s: Name of the block
+		const titleFormat = Platform.OS === 'android' ? __( '\'%s\' isn\'t yet supported on WordPress for Android' ) :
+			__( '\'%s\' isn\'t yet supported on WordPress for iOS' );
```

Comment:
Not an issue for this PR, but noting for the future:  I just realized this might be used in other apps, so we should find  a way to inject the app name from the apps instead.

Reply:
Good point!